vanilla-framework/vanilla-framework
# Tidy up local development tooling

## Done
Added `./run` to the project. Integrate the npm scripts with `./run`. Updated the README and CONTRIBUTING docs with the new commands.

## QA
- Pull this branch
- Run `./run clean` and check it removes all the build artefacts
- Run `./run` and check it sets up the project in a container with all its dependencies and the site works
- Run `./run watch` and make a change to the Sass and check after a refresh the change is visible
- Run `./run test` and check that runs the tests

## Details
Fixes https://github.com/vanilla-framework/vanilla-framework/issues/1075
